java/core/pom.xml

```diff
@@ -11,10 +11,39 @@
     </parent>
 
     <groupId>co.worklytics.psoxy</groupId>
-    <artifactId>psoxy-core</artifactId>
+    <artifactId>psoxy-core${artifactSuffix}</artifactId>
     <packaging>jar</packaging>
     <description>Code that's core to both Psoxy clients and implementations</description>
 
+    <properties>
+        <!-- empty for java 11 builds -->
+        <artifactSuffix></artifactSuffix>
```

Comment:
I'd prefer to follow always same convention: `psoxy-core-jdkXX` instead of keeping it empty for jdk11

Reply:
I'm OK with it empty ... jdk8 is really a special case for use; I prefer to keep it simple for users - they don't need to consider what jdk version they're building for, it should be implementation detail